### PR TITLE
Add NFTs empty state

### LINKED ISSUE #2624: Add empty state of the NFTs page
- [ ] add header
- [ ] add description
- [ ] "Add account" link
- [ ] Add "explore markets" banners to the tabs

image

### PR DESCRIPTION
**Pull request by hyphenized**
Closes #2624

## Testing Env

```
SUPPORT_NFT_TAB=true
```

## To Test
- [ ] Remove all accounts, the header with the empty state should be visible

Latest build: [extension-builds-2669](https://github.com/tallycash/extension/suites/9505204669/artifacts/451023010) (as of Fri, 25 Nov 2022 12:59:25 GMT).

### CODE REVIEW

**Comment by hyphenized** [comment]
@VladUXUI please check the design, ignore the casing in the tab's title, it's related to a bug being addressed in #2664

**Comment by jagodarybacka** [comment]
@VladUXUI I think buttons should be "OpenSea" and "LooksRare" as that's how they have it on their logos

**Review by jagodarybacka — CHANGES_REQUESTED** [review]
Very nice ✨ Couple things I noticed + can we get marketplace icons in about the same size? it doesn't matter that much here probably as they are very small anyway but would be nice

**Comment by VladUXUI** [comment]
> @VladUXUI please check the design, ignore the casing in the tab's title, it's related to a bug being addressed in #2664

looks good! need to separate them into tabs though.

**Comment by VladUXUI** [comment]
> @VladUXUI I think buttons should be "OpenSea" and "LooksRare" as that's how they have it on their logos

agreed! thanks for noticing

File: ui/components/NFTS_update/NFTsExploreBanner.tsx [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+]
+
+type ExporeLinkProps = {
+  url: string
+  title: string
+  color: string
+  icon: string
+}
+
+function ExploreLink({
```
[the remark this answers is not in the document]
Comment (VladUXUI) [reply ui/components/NFTS_update/NFTsExploreBanner.tsx:36]: For this instance, let's not use a button style. As the CTA's become tangled a bit.
But i'm ok if we use current design with icon on the right side and no border.

My bad for not being consistent with placement of icon/logo

File: ui/components/NFTS_update/ExploreMarketLink.tsx [hunk 2]
```diff
@@ -0,0 +1,50 @@
+import React from "react"
+
+type ExporeMarketLinkProps = {
```
Comment (greg-nagy) [inline ui/components/NFTS_update/ExploreMarketLink.tsx:3]: typo

**Review by greg-nagy — CHANGES_REQUESTED** [review]
this is looking really good 🤩

The topbar should not be visible here. I think the #2664 broke [its hidden mechanism](https://github.com/tallycash/extension/blob/e4a46d75b7490073f64b54d460672cbffbb8bc7f/ui/pages/Popup.tsx#L211), but haven't played it out fully. 

CleanShot 2022-11-25 at 10 49 22

The other blocker is the has no nft vs has no account change.

File: ui/pages/NFTs.tsx [hunk 3]
```diff
@@ -1,24 +1,31 @@
...
-import SharedBanner from "../components/Shared/SharedBanner"
+import NFTsExploreBanner from "../components/NFTS_update/NFTsExploreBanner"
+import NFTsHeader from "../components/NFTS_update/NFTsHeader"
+import { useBackgroundSelector } from "../hooks"
 
 export default function NFTs(): ReactElement {
-  const { t } = useTranslation("translation", {
-    keyPrefix: "nfts",
-  })
+  const hasAccounts = useBackgroundSelector(getHasAccounts)
```
[the remark this answers is not in the document]
Comment (hyphenized) [reply ui/pages/NFTs.tsx:8]: Ooops! You're right! Thanks for catching that!

File: ui/components/NFTS_update/ExploreMarketLink.tsx (hunk 2 above)
[the remark this answers is not in the document]
Comment (hyphenized) [reply ui/components/NFTS_update/ExploreMarketLink.tsx:3]: Thanks! Nice extension 👀

**Comment by hyphenized** [comment]
> this is looking really good 🤩
> 
> The topbar should not be visible here. I think the #2664 broke [its hidden mechanism](https://github.com/tallycash/extension/blob/e4a46d75b7490073f64b54d460672cbffbb8bc7f/ui/pages/Popup.tsx#L211), but haven't played it out fully.
> 
> CleanShot 2022-11-25 at 10 49 22
> 
> The other blocker is the has no nft vs has no account change.

Hmmm I can't get the topbar to appear, can you try switching between tabs and see if that clears the bug 🤔 ?

**Comment by greg-nagy** [comment]
> > this is looking really good 🤩
> > The topbar should not be visible here. I think the #2664 broke [its hidden mechanism](https://github.com/tallycash/extension/blob/e4a46d75b7490073f64b54d460672cbffbb8bc7f/ui/pages/Popup.tsx#L211), but haven't played it out fully.
> > CleanShot 2022-11-25 at 10 49 22
> > The other blocker is the has no nft vs has no account change.
> 
> Hmmm I can't get the topbar to appear, can you try switching between tabs and see if that clears the bug 🤔 ?

hm, it was a really weird behavior on my end — couple of restarts and reinstalls has fixed the issue

**Review by greg-nagy — APPROVED** [review]
this is ready to be merged ⚒

**Pull request merged**